src/recipe/__mocks__/recipe.controller.mock.ts

```diff
@@ -16,25 +16,63 @@ export class MockRecipeController {
       id: faker.number.int(),
       createdAt: new Date(),
       ...createRecipeRequest,
+      isPublic: true,
+      authorId: faker.number.int(),
     };
 
     return Promise.resolve(CreateRecipeResponse.from(createdRecipe));
   }
 
-  fetchRecipe(id: number): Promise<FetchRecipeResponse> {
+  fetchRecipe(
+    req: { userId: number },
+    recipeId: number,
+  ): Promise<FetchRecipeResponse> {
     const fetchedRecipe = {
-      id,
+      id: recipeId,
       createdAt: new Date(),
       title: faker.word.noun(),
       description: faker.lorem.text(),
       ingredients: faker.lorem.words(4),
       preparation: faker.lorem.lines(5),
+      isPublic: true,
+      authorId: req.userId,
     };
 
     return Promise.resolve(FetchRecipeResponse.from(fetchedRecipe));
   }
 
-  fetchAllRecipes(): Promise<FetchRecipesResponse> {
+  fetchUsersRecipes(
+    _req: { userId: number },
+    _email: string,
+  ): Promise<FetchRecipesResponse> {
+    const authorId = faker.number.int();
+    const fetchedUsersRecipes = [
+      {
+        id: 0,
+        createdAt: new Date(),
+        title: faker.word.noun(),
+        description: faker.lorem.text(),
+        ingredients: faker.lorem.words(4),
+        preparation: faker.lorem.lines(5),
+        isPublic: true,
+        authorId,
+      },
+      {
+        id: 1,
+        createdAt: new Date(),
+        title: faker.word.noun(),
+        description: faker.lorem.text(),
+        ingredients: faker.lorem.words(4),
+        preparation: faker.lorem.lines(5),
+        isPublic: true,
+        authorId,
+      },
```

Comment:
I would add some kind of method to generate recipes for mocks and tests :
something like: generateRecipeResponse(): RecipeResponse